environment.yml

```
@@ -1,6 +1,6 @@
 # You can use this file to create a conda environment for this pipeline:
 #   conda env create -f environment.yml
-name: nf-core-mhcquant-1.5.1
+name: nf-core-mhcquant-1.6
```

```suggestion
name: nf-core-mhcquant-1.6.0
```

The versions should follow semantic versioning, so you can bump versions again to `1.6.0`

---

Actually nf-core tools should in my opinion just not allow versions that do not adhere to semantic versioning.
It's an easy check.